graphics/asymptote/Portfile

```diff
@@ -29,13 +28,14 @@ set python.bin      ${prefix}/bin/python${python.branch}
 master_sites        sourceforge:project/${name}/${version}/
 extract.suffix      .src.tgz
 
-checksums           rmd160  c78a5d06d17dfda25e785e7b599e044e6374494b \
-                    sha256  f5cc913a858c33e92f79ab421d354c0fe2babd87f452ae9dff729a902aa80c3f \
-                    size    4674495
+checksums           rmd160  46422bdc2c0fc99b51386333039d80cba0b2a0a0 \
+                    sha256  4f6fbc5aa058c8d38513546be5b233bb5d7fa50e39486f3d8624aa7595ae748d \
+                    size    15070290
 
 # On Lion the default clang fails with
 # ../base/graph_splinetype.asy: 13.10: arrays have different lengths: 24 != 25
-compiler.blacklist  {clang < 500}
+# gcc-4.2 fails on 10.6 with multiple errors.
+compiler.blacklist  {clang < 500} *gcc-4.*
```

Comment:
According to the author, LSP probably requires
```
compiler.cxx_standard   2014
```
which most likely covers gcc 4 exclusion as well.

Can you please test that?

(I wonder if we should make LSP optional or not.)

Reply:
Yes, it does exclude `gcc-4.2`, but the build still fails on generating some pdf docs (some do generate correctly though):
```
cd doc && /usr/bin/make man
cd doc && /usr/bin/make faq
Creating asy-keywords.el
./asy -dir base -config "" -render=0 -l > asy.list
make[1]: Entering directory `/opt/local/var/macports/build/_opt_PPCRosettaPorts_graphics_asymptote/asymptote/work/asymptote-2.82/doc'
make[1]: `faq' is up to date.
make[1]: Leaving directory `/opt/local/var/macports/build/_opt_PPCRosettaPorts_graphics_asymptote/asymptote/work/asymptote-2.82/doc'
make[1]: Entering directory `/opt/local/var/macports/build/_opt_PPCRosettaPorts_graphics_asymptote/asymptote/work/asymptote-2.82/doc'
../asy -dir ../base -config "" -render=0 -h 2>&1 | grep -iv Asymptote > options
../asy -dir ../base -config "" -render=0 -f pdf -noprc Bode.asy
../asy -dir ../base -config "" -render=0 -f pdf -noprc CAD1.asy
../asy -dir ../base -config "" -render=0 -f pdf -noprc CDlabel.asy
../asy -dir ../base -config "" -render=0 -f pdf -noprc GaussianSurface.asy
make[1]: *** [options] Error 1
make[1]: *** Waiting for unfinished jobs....
/bin/sh: line 1:  3798 Bus error               ./asy -dir base -config "" -render=0 -l > asy.list
make: *** [asy-keywords.el] Error 138
make: *** Waiting for unfinished jobs....
make[1]: *** [CAD1.pdf] Bus error
make[1]: *** [Bode.pdf] Bus error
make[1]: *** [CDlabel.pdf] Bus error
make[1]: *** [GaussianSurface.pdf] Bus error
make[1]: Leaving directory `/opt/local/var/macports/build/_opt_PPCRosettaPorts_graphics_asymptote/asymptote/work/asymptote-2.82/doc'
make: *** [man] Error 2
```

Reply:
@mojca Yeah, it seems to be completely broken when built with Clang. Opened an issue with upstream: https://github.com/vectorgraphics/asymptote/issues/346